### DISCUSSION TITLE
Is Barrier still maintained?

### DISCUSSION THREAD

**Discussion by n-sweep** [debauchee/barrier#1892] [General]
I've been using it for so long that it's become a necessary part of my workspace. If you've used/moved to Synergy or recently made the move from Synergy to Barrier, I'd love to hear your perspective.

**Comment by lexxxel** [comment]
Look like some active developers moved away from this repo to https://github.com/input-leap/input-leap

**Reply by shymega** [reply]
We weren't just 'some' - we were the *only* active maintainers. I keep an eye on the discussions tab of Barrier. We're currently working towards a v3.0.0 release, and tidying up the codebase. An initial Wayland backend has been added as well.

**Reply by lexxxel** [reply]
I didn't want to offend you. I haven't checked if you were the only 2 people writting commits, that's why I phrased it: some developers, not some maintainers.
I was super happy that I found barrier, but it toke me a while to find input-leap. That's why I decided to link to your project... that was the import part of my post. The rest were just necessary words to frame that 😉

**Reply by n-sweep** [reply]
Looks like I found my new tool. Thanks to you both

**Reply by Acanis87** [reply]
Thanks! :) Good to know

**Reply by shymega** [reply]
> I didn't want to offend you. I haven't checked if you were the only 2 people writting commits, that's why I phrased it: some developers, not some maintainers. I was super happy that I found barrier, but it toke me a while to find input-leap. That's why I decided to link to your project... that was the import part of my post. The rest were just necessary words to frame that wink

Not offended whatsoever. Just correcting you. There's a fine distinction between developers and maintainers. Example: we have PRs coming in from outside contributors, but they're not maintainers.

**Reply by aphix** [reply]
If somebody is using older (e.g. win10) laptops/desktops, what would you recommend them starting with at this point to get basic shared kb/m functionality with a single host machine (assuming they are using nothing currently)?

**Reply by shymega** [reply]
@aphix My advice would be to keep an eye on Input Leap for new releases, and for now, to use Barrier v2.3.4 at the **least**, or v2.4.0. Bear in mind Barrier is in a state where bugs won't be fixed.

**Reply by aphix** [reply]
Cheers, thanks for the quick response!

**Reply by nbolton** [reply]
Barrier postmortem: [Why did Barrier fail?](https://symless.com/synergy/news/why-did-barrier-fail)

**Comment by nbolton** [comment]
Try [Deskflow](https://github.com/deskflow/deskflow), the upstream of Synergy, Input Leap, and Barrier.
